apps/etterlatte-utbetaling/src/main/kotlin/utbetaling/iverksetting/utbetaling/Utbetaling.kt

```diff
@@ -67,9 +67,26 @@ data class Utbetalingslinje(
     val sakId: SakId,
     val periode: PeriodeForUtbetaling,
     val beloep: BigDecimal? = null,
-    val klassifikasjonskode: OppdragKlassifikasjonskode
+    val klassifikasjonskode: OppdragKlassifikasjonskode,
+    val brukKjoereplan: BrukKjoereplan
 )
 
+enum class BrukKjoereplan(private val kode: String) {
```

Comment:
Hvorfor ikke bare `Kjoereplan`? "Bruk kjøreplan" gir inntrykk av at det er en boolean.

Reply:
Etter litt graving i koden ser jeg at dette stammer fra oppdrag. Tenker vi allikevel kan omdøpe litt på vår side. 

Denne enumklassen kunne også med fordel hatt en liten kommentar tilknyttet kodene `J` og `N`, samt en test som sjekker at det ikke blir tuklet med.

Reply:
Bare et eller annet som gjør J / N litt mer forståelig for nestemann som skal inn i koden. 

Eksempelvis har vi lagt til dette i brevkoden hvor vi journalfører til Dokarkiv: 
```
            tema = "EYB", // https://confluence.adeo.no/display/BOA/Tema
            kanal = "S", // https://confluence.adeo.no/display/BOA/Utsendingskanal
```